docs/how-to-work-on-coding-challenges.md

```diff
@@ -276,8 +276,8 @@ The following is an example of code:
 ```
 ````
 
-- Additional information in the form of a note should be formatted `**Note:** Rest of note text...`
-- If multiple notes are needed, then list all of the notes in separate sentences using the format `**Notes:** First note text. Second note text.`.
+- Additional information in the form of a note should be separated by a newline, and formatted: `**Note:** Rest of note text...`
```

Comment:
```suggestion
- Additional information in the form of a note should be surrounded by blank lines, and formatted: `**Note:** Rest of note text...`
```
Hey Shaun, did you mean a note should be like this:

```
optional stuff before followed by an empty line

**Note:** Rest of note text...

optional stuff after preceded by an empty line
```

If so, what do you think to this wording?

Reply:
That is clearer 👍